EnviroMonitorFrontend/package.json

```diff
@@ -7,8 +7,9 @@
     "test": "mocha --compilers js:babel-core/register tests/**/*.spec.js",
     "test:watch": "npm run test -- --watch",
     "coverage": "nyc --reporter text --reporter text-summary --reporter html npm run test",
-    "start": "webpack-dev-server",
-    "build": "webpack"
+    "start": "webpack-dev-server --content-base ./build/",
+    "build": "webpack",
+    "build:watch": "webpack --progress --colors --watch"
```

Comment:
@ajakubo1 

1. I've tried to run docker image only with server instance (that runs `npm run start`) and any change in react file was reflected on page (automatic reload). So I assume yes. So I'll remove build:watch command for now - ok?
2. I  haven't found any watch mode for nyc but I weren't searching much...